cmake/modules/extensions.cmake

```diff
@@ -4044,15 +4026,10 @@ function(zephyr_linker_section_configure)
     endif()
   endif()
 
-  set(SECTION)
-  zephyr_linker_arg_val_list(SECTION "${single_args}")
-  zephyr_linker_arg_val_list(SECTION "${options}")
-  zephyr_linker_arg_val_list(SECTION "${multi_args}")
+  zephyr_linker_add_property(SECTION "${single_args}" SECTION_SETTINGS)
+  zephyr_linker_add_property(SECTION "${options}" SECTION_SETTINGS)
+  zephyr_linker_add_property(SECTION "${multi_args}" SECTION_SETTINGS)
```

Comment:
The original value for `SECTION_SETTINGS` property is `{SECTION;.intList;ANY;FALSE;FIRST;TRUE;KEEP;TRUE;INPUT;.irq_info}`.
Now the value changed to `{SECTION;.intList}{ANY;FALSE;FIRST;TRUE;KEEP;TRUE}{INPUT;.irq_info}`.

Not sure if this can cause problems. Still thinking about it.